common/api/core-common.api.md

```diff
@@ -1,8955 +1,8955 @@
-## API Report File for "@itwin/core-common"
```

Comment:
Any idea why this file is completely flagged as changing? Whitespace/line endings?

Reply:
nope, it's an auto-generated file.

Reply:
FYI I use [this extension](https://chrome.google.com/webstore/detail/github-whitespace/fnpkdafamnbjoldglihkjjdicofghccm?hl=en-US) to force github diffs to ignore whitespace by default; otherwise you constantly have to click that option and reload the page - github doesn't have a way to save this as a preference.